api/src/test/java/org/openmrs/module/locationbasedaccess/aop/UserSearchAdviserTest.java

```diff
@@ -0,0 +1,617 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.locationbasedaccess.aop;
+
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import org.junit.Test;
+import org.openmrs.Location;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.openmrs.Person;
+import org.openmrs.User;
+import org.openmrs.Role;
+import org.openmrs.api.PersonService;
+import org.openmrs.api.UserService;
+import org.openmrs.api.LocationService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.locationbasedaccess.aop.common.AOPContextSensitiveTest;
+import org.openmrs.module.locationbasedaccess.aop.common.TestWithAOP;
+import org.openmrs.module.locationbasedaccess.aop.interceptor.UserServiceInterceptorAdvice;
+
+public class UserSearchAdviserTest extends AOPContextSensitiveTest {
+    
+    private UserService userService;
+    private PersonService personService;
+    private LocationService locationService;
+    private static final String XML_FILENAME_WITH_ADMIN_USER_DATA = "include/AdminUserData.xml";
+    private static final String XML_FILENAME_WITH_PERSON_DATA = "include/PatientTestData.xml";
+    private static final String DEMO_USER_PROPERTY= "locationUuid";

```

Comment:
Can you just rename it to DEMO_LOCATION_USER_PROPERTY?

Reply:
Made the necessary changes.